website/project/views/drafts.py

```diff
@@ -121,6 +123,9 @@ def submit_draft_for_review(auth, node, draft, *args, **kwargs):
     :rtype: dict
     :raises: HTTPError if embargo end date is invalid
     """
+    if waffle.switch_is_active(features.OSF_PREREGISTRATION):
```

Comment:
Can you add a test to ensure that this function doesn't execute when the switch is active?

Reply:
👍